

drivers/net/Kconfig:1749:warning: 'select' used by config symbol 'VIA_VELOCITY'

We renamed CONFIG_CRC16.

Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/drivers/net/Kconfig |    2 +-
 1 files changed, 1 insertion(+), 1 deletion(-)

diff -puN drivers/net/Kconfig~net-kconfig-crc16-fix drivers/net/Kconfig
--- 25/drivers/net/Kconfig~net-kconfig-crc16-fix	2004-07-11 20:34:19.556008664 -0700
+++ 25-akpm/drivers/net/Kconfig	2004-07-11 20:34:19.561007904 -0700
@@ -1746,7 +1746,7 @@ config VIA_VELOCITY
 	tristate "VIA Velocity support"
 	depends on NET_PCI && PCI
 	select CRC32
-	select CRC16
+	select CRC_CCITT
 	select MII
 	help
 	  If you have a VIA "Velocity" based network card say Y here.
_
